BiweeklyMeetings/2022-06-07.md

```diff
@@ -0,0 +1,370 @@
+
+**Matthias** -: And the stake on the test net would be for instance, much more different because it's all free money and people don't have the same behavior, therefore. So, yeah, I don't know. We can try to bring IOG back again, see what's the answer and keep to have short update on that every week or every two weeks.
+
+**M. Àngels** -:  Okay. So we are going to keep it through for next meeting agenda?
+
+**Matthias** -: I would say for now, yes. And let's see, we probably won't get any update from put outputs before the Hard Fork. Because I guess all minds are focused on that, but we might a bit later.
+
+**M. Àngels** -:  Okay. So that was all that was in the agenda for today's meeting. So maybe, Matthias, you want to review those new CIP you commented at the beginning of the session?
+
+**Matthias** -: Yes. But I just have a small urgency to cope with, so I will be back in two minutes.
+
+**M. Àngels** -:  Okay.
+
+
+### Discussions
+
+#### PR267
+ [PR267 CIP-0025: Added version 2 and cddl](https://github.com/cardano-foundation/CIPs/pull/267)
+
+**Matthias** -: But I would like to look at PR 267 CIP-0025: Added version 2 and cddl, at least. So maybe we can already print to screen.
+
+**M. Àngels** -:  Sure. Take your time, Matthias and I will leave it here for everybody to read it. I will share the link on the chat and also here. So let's wait a few minutes until Matthias is back.
+
+**Matthias** -: Okay. Sorry for that.
+
+**M. Àngels** -:  No problem. Do you want to make a brief summary of CIP 267?
+
+**Matthias** -: CIP-0025: Added version 2 and cddl #267
+
+**Matthias** -: No, sorry. Yeah. Pull Request 267. Yeah. So this is a iteration on CIP 25, right? Which was the NFT metadata standard we ha not exposed, but mentioned a few points that could be improved last time. And the idea was to make a version two that would address those points or this is basically it. Okay. What I see here. Yeah. I think one thing we discussed and that's also Sebastian maybe mentioning it, or. So it would be good to have the CGTL as a separate files as always, but at least now we've got a clear onchain CDDL specifications of the metadata for the CIP.
+
+**Matthias** -: I think it is mainly compatible version one in the sense that it embeds the same informations, but there is a slight difference in the way the asset names gets included. For instance, right now it used to be just playing UTF-8 text string, whereas this new one uses actually raw binary bites through CBO coding. So since this is entry data any way, that makes much more sense.
```

Comment:
```suggestion
**Matthias** -: I think it is mainly compatible version one in the sense that it embeds the same informations, but there is a slight difference in the way the asset names gets included. For instance, right now it used to be just playing UTF-8 text string, whereas this new one uses actually raw binary bytes through CBOR encoding. So since this is entry data any way, that makes much more sense.
```